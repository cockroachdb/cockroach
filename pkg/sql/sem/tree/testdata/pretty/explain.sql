SELECT "Level", "Type" FROM [EXPLAIN (VERBOSE) SELECT * FROM kv AS a JOIN kv USING (k) WHERE a.v > 3 ORDER BY a.v DESC]
