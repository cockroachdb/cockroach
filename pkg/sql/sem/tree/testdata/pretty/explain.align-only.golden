// Code generated by TestPretty. DO NOT EDIT.
// GENERATED FILE DO NOT EDIT
1:
-
SELECT
	"Level",
	"Type"
FROM
	[
		EXPLAIN (
			VERBOSE
		)
			SELECT
				*
			FROM
				kv
					AS a
				JOIN
					kv
				USING (k)
			WHERE
				a.v
				> 3
			ORDER BY
				a.v
					DESC
	]

15:
---------------
SELECT "Level",
       "Type"
  FROM [
		EXPLAIN (
			VERBOSE
		)
			SELECT
				*
			FROM
				kv
					AS a
				JOIN
					kv
				USING (k)
			WHERE
				a.v
				> 3
			ORDER BY
				a.v
					DESC
       ]

17:
-----------------
SELECT "Level",
       "Type"
  FROM [
		EXPLAIN (
			VERBOSE
		) SELECT
			*
		  FROM
			kv
				AS a
			JOIN
				kv
			USING (k)
		  WHERE
			a.v
			> 3
		  ORDER BY
			a.v
				DESC
       ]

19:
-------------------
SELECT "Level",
       "Type"
  FROM [
		EXPLAIN (
			VERBOSE
		) SELECT
			*
		  FROM
			kv AS a
			JOIN kv
			USING (k)
		  WHERE
			a.v > 3
		  ORDER BY
			a.v
				DESC
       ]

20:
--------------------
SELECT "Level",
       "Type"
  FROM [
		EXPLAIN (
			VERBOSE
		)   SELECT *
		      FROM kv
					AS a
		           JOIN
					kv
		           USING (k)
		     WHERE a.v
		           > 3
		  ORDER BY a.v
					DESC
       ]

22:
----------------------
SELECT "Level", "Type"
  FROM [
		EXPLAIN (
			VERBOSE
		)   SELECT *
		      FROM kv
					AS a
		           JOIN
					kv
		           USING (k)
		     WHERE a.v
		           > 3
		  ORDER BY a.v
					DESC
       ]

26:
--------------------------
SELECT "Level", "Type"
  FROM [
		EXPLAIN (
			VERBOSE
		)   SELECT *
		      FROM kv AS a
		           JOIN kv
		           USING (k)
		     WHERE a.v > 3
		  ORDER BY a.v
					DESC
       ]

27:
---------------------------
SELECT "Level", "Type"
  FROM [
		EXPLAIN (
			VERBOSE
		)   SELECT *
		      FROM kv AS a
		           JOIN kv
		           USING (k)
		     WHERE a.v > 3
		  ORDER BY a.v DESC
       ]

32:
--------------------------------
SELECT "Level", "Type"
  FROM [
		EXPLAIN (VERBOSE) SELECT
							*
		                  FROM
							kv
								AS a
							JOIN
								kv
							USING (k)
		                  WHERE
							a.v
							> 3
		                  ORDER BY
							a.v
								DESC
       ]

35:
-----------------------------------
SELECT "Level", "Type"
  FROM [
		EXPLAIN (VERBOSE) SELECT
							*
		                  FROM
							kv AS a
							JOIN kv
							USING (k)
		                  WHERE
							a.v > 3
		                  ORDER BY
							a.v
								DESC
       ]

36:
------------------------------------
SELECT "Level", "Type"
  FROM [
		EXPLAIN (VERBOSE)   SELECT *
		                      FROM kv
									AS a
		                           JOIN
									kv
		                           USING (k)
		                     WHERE a.v
		                           > 3
		                  ORDER BY a.v
									DESC
       ]

42:
------------------------------------------
SELECT "Level", "Type"
  FROM [
		EXPLAIN (VERBOSE)   SELECT *
		                      FROM kv AS a
		                           JOIN kv
		                           USING (k)
		                     WHERE a.v > 3
		                  ORDER BY a.v
									DESC
       ]

43:
-------------------------------------------
SELECT "Level", "Type"
  FROM [
		EXPLAIN (VERBOSE)   SELECT *
		                      FROM kv AS a
		                           JOIN kv
		                           USING (k)
		                     WHERE a.v > 3
		                  ORDER BY a.v DESC
       ]

52:
----------------------------------------------------
SELECT "Level", "Type"
  FROM [
		EXPLAIN (VERBOSE)   SELECT *
		                      FROM kv AS a
		                           JOIN kv USING (k)
		                     WHERE a.v > 3
		                  ORDER BY a.v DESC
       ]

60:
------------------------------------------------------------
SELECT "Level", "Type"
  FROM [
		EXPLAIN (VERBOSE)   SELECT *
		                      FROM kv AS a JOIN kv USING (k)
		                     WHERE a.v > 3
		                  ORDER BY a.v DESC
       ]

97:
-------------------------------------------------------------------------------------------------
SELECT "Level", "Type"
  FROM [
		EXPLAIN (VERBOSE) SELECT * FROM kv AS a JOIN kv USING (k) WHERE a.v > 3 ORDER BY a.v DESC
       ]

98:
--------------------------------------------------------------------------------------------------
SELECT "Level", "Type"
  FROM [EXPLAIN (VERBOSE) SELECT * FROM kv AS a JOIN kv USING (k) WHERE a.v > 3 ORDER BY a.v DESC]

119:
-----------------------------------------------------------------------------------------------------------------------
SELECT "Level", "Type" FROM [EXPLAIN (VERBOSE) SELECT * FROM kv AS a JOIN kv USING (k) WHERE a.v > 3 ORDER BY a.v DESC]


